homeassistant/components/agent_dvr/camera.py

```diff
@@ -80,11 +80,11 @@ def __init__(self, device):
         """Initialize as a subclass of MjpegCamera."""
         self.device = device
         self._removed = False
-        self._attr_unique_id = f"{device._client.unique}_{device.typeID}_{device.id}"
+        self._attr_unique_id = f"{device.client.unique}_{device.typeID}_{device.id}"
```

Comment:
is this intentional?

Reply:
Yes, I looked into the library and they are the same (the lines 86 and 87 access it as this as well)